src/main/java/seedu/stocker/commands/DeleteCommand.java

```diff
@@ -0,0 +1,44 @@
+package seedu.stocker.commands;
+
+import seedu.stocker.drugs.StockEntry;
+import seedu.stocker.exceptions.DrugNotFoundException;
+
+public class DeleteCommand  extends Command{
+
+    public static final String COMMAND_WORD = "delete";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Removes a drug from drug list. "
+            + "Parameters: Name  " + System.lineSeparator()
+            + "Example: " + COMMAND_WORD
+            + " <Drug Name>";
+
+    public static final String MESSAGE_SUCCESS = "Drug removed from inventory: %1$s";
+    public static final String MESSAGE_FAILURE = "Drug not find in the inventory. ";
+
+
+
+    private final String keyToDelete;
+
+    /**
+     * Constructs a DeleteCommand with the specified drug name.
+     *
+     */
+    public DeleteCommand(String key) {
+        this.keyToDelete = key.trim().toLowerCase();
+    }
```

Comment:
Can consider using clearer variable names, I would assume key is more affiliated  with a HashMap.